packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadCard.tsx

```diff
@@ -111,6 +112,16 @@ export const ThreadCard = ({
     (thread.stage && !isStageDefault) || linkedProposals?.length > 0;
   const stageLabel = threadStageToLabel(thread.stage);
 
+  // Future Ref: this fixes https://github.com/hicommonwealth/commonwealth/issues/8611 for iOS mobile
+  // where quill renders broken/cut-off/overlapping thread.plaintext in cases when there are multiple
+  // <p/> tags in the quill delta for thread.plaintext or if thread.plaintext has \n characters which
+  // iOS devices don't seem to render correctly.
+  // Not updating it for desktop per a previous issue where markdown wasn't rendered correctly in
+  // preview because of .slice()'d  content.
```

Comment:
i am not a fan of these long comments, but left it there as we keep getting quill related issues